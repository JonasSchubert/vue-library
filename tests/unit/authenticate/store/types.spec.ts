import {
  ActionTypes, GetterTypes, ModuleType, MutationTypes, RouteTypes
} from '@/authenticate/store/types';

describe('ActionTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(ActionTypes).toMatchSnapshot();
  });
});

describe('GetterTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(GetterTypes).toMatchSnapshot();
  });
});

describe('ModuleType', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(ModuleType).toMatchSnapshot();
  });
});

describe('MutationTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(MutationTypes).toMatchSnapshot();
  });
});

describe('RouteTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(RouteTypes).toMatchSnapshot();
  });
});
