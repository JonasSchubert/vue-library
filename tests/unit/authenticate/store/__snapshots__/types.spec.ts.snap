// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionTypes should not change unexpected 1`] = `
Object {
  "login": "login",
  "logout": "logout",
  "validateToken": "validate-token",
}
`;

exports[`GetterTypes should not change unexpected 1`] = `
Object {
  "error": "error",
  "isAdministrator": "isAdministrator",
  "isLoading": "isLoading",
  "isUser": "isUser",
  "loggedIn": "loggedIn",
  "token": "token",
  "userRoles": "userRoles",
}
`;

exports[`ModuleType should not change unexpected 1`] = `"authenticate"`;

exports[`MutationTypes should not change unexpected 1`] = `
Object {
  "setError": "setError",
  "setIsLoading": "setIsLoading",
  "setToken": "setToken",
  "setUserRoles": "setUserRoles",
}
`;

exports[`RouteTypes should not change unexpected 1`] = `
Object {
  "authenticate": "authenticate",
}
`;
